### PR TITLE
feat(#2w79jf_#4p6r5f): add Logo Component

### PR DESCRIPTION
**Pull request by briankimlim**

### CODE REVIEW

**Review by thien-ng — APPROVED** [review]

**Review by asianswagstorm — APPROVED** [review]

File: src/components/Logo/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './styles.scss';
+
+export const Logo = () => {
+    return (
+        //TODO: change srcSet urls to 'src/assets/logos' image assets
+        <picture>
+            <source media="(min-width: 768px)" srcSet='https://via.placeholder.com/60x60.png?text=60x60+Logo'/>
```
Comment (alphakennyn) [inline src/components/Logo/Logo.jsx]: For the small logo, your `media` should consider the height not the width as the navbar (where the small one is going) will still share the same width as the page

File: src/components/Logo/Logo.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './styles.scss';
+
+export const Logo = () => {
+    return (
+        //TODO: change srcSet urls to 'src/assets/logos' image assets
+        <picture>
```
Comment (alphakennyn) [inline src/components/Logo/Logo.jsx]: nice!

File: src/components/Logo/__tests__/Logo.test.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Logo } from '../Logo.jsx';
+
+describe('Logo', () =>{
+    describe('render', () => {
+        it('renders without crashing', () => {
+            const div = document.createElement('div');
```
Comment (alphakennyn) [inline src/components/Logo/__tests__/Logo.test.jsx]: please replace with snapshot tests (See login tests for example)

File: src/views/App/styles.scss [hunk 4]
```diff
@@ -4,18 +4,18 @@
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-}
-
 .App-header {
```
Comment (alphakennyn) [inline src/views/App/styles.scss:7]: Just note that once @Kevinjhlin navbar branch is merged, then this class won't matter anymore

**Review by alphakennyn — CHANGES_REQUESTED** [review]

**Review by Kevinjhlin — APPROVED** [review]

File: src/components/Logo/Logo.jsx (hunk 1 above)
Comment (briankimlim) [reply src/components/Logo/Logo.jsx]: Oh I thought the Logo in this case was not in the Navbar, but in the Header above Navbar. What dimensions are appropriate in that case for height? I used 3 dimensions sizes for mobile, tablet & desktop so far but I'm not sure what's appropriate. 

The logo size in the Header is changing depending on what device I use at least.

Comment (alphakennyn) [reply src/components/Logo/Logo.jsx]: I was thinking the Logo component would be reused in the Landing page and in the navbar, which should ultimately resize

We won't need different image source for different device (Except for desktop where we'd need something bigger)

**Review by thien-ng — APPROVED** [review]

**Pull request merged**